packager/packager.cc

```diff
@@ -243,6 +243,50 @@ class PackageInfoImpl final : public sorbet::core::packages::PackageInfo {
               fmt::format("\n  {} {}", isTestImport ? "test_import" : "import", info.name.toString(gs))}});
         return {suggestion};
     }
+
+    std::optional<core::AutocorrectSuggestion>
+    addExport(const core::GlobalState &gs, const vector<core::NameRef> newExport, bool isPrivateTestExport) const {
+        for (auto expt : exports) {
+            // check if we already import this, and if so, don't
+            // return an autocorrect
+            if (expt.fqn.parts == newExport) {
+                return nullopt;
+            }
+        }
+
+        core::Loc insertionLoc = loc.adjust(gs, core::INVALID_POS_LOC, core::INVALID_POS_LOC);
```

Comment:
this adds `INVALID_POS_LOC` to the beginPos and endPos.

Do you maybe want to do something like.

but you also write to this variable in every branch. do you just want to do

```suggestion
        auto insertionLoc = core::Loc::none(loc.file());
```

?